EXOSIMS/Prototypes/TargetList.py

```diff
@@ -159,7 +161,7 @@ class TargetList(object):
             if attribute ``getKnownPlanets`` is True. Otherwise all entries are False.
         Hmag (numpy.ndarray):
             H band magnitudes
-        I (astropy.units.quantity.Quantity):
+        systemInclination (astropy.units.quantity.Quantity):

```

Comment:
I would like for class attributes to be in alphabetical order in the docstrings (inputs should be in the input order, of course). Makes the compiled docs easier to search.